test/unit/connection_string.test.ts

```diff
@@ -291,9 +291,10 @@ describe('Connection String', function () {
 
   describe('resolveSRVRecord()', () => {
     const resolveSRVRecordAsync = promisify(resolveSRVRecord);
+    const sandbox = sinon.createSandbox();
```

Comment:
The unit tests I added in get_more.test.ts started failing when I ran the full test suite.  It seems like these tests were clear the mock I was setting in `it('executes a getMore on the provided server')`.  The solution is to use a sinon sandbox.

Reply:
I refactored the get_more unit tests so these changes are no longer necessary